src/components/CommentUsername/CommentUsername.js

```diff
@@ -25,7 +25,7 @@ const styles = StyleSheet.create({
   dot: {
     fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(12),
-    color: '#828282'
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the dot has been changed from `'#828282'` to `COLORS.gray400`. This seems like a good change as it uses a predefined constant instead of a hardcoded value, improving maintainability.
```diff
-    color: '#828282'
+    color: COLORS.gray400
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->